index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
```

Comment:
If we not put `!Doctype` in html so what will happen ???

Reply:
we won't be able to render the web properly and also it will inform the browser which version of html we are using.